dataland-backend/src/main/kotlin/org/dataland/datalandbackend/controller/EuTaxonomyDataForNonFinancialsController.kt

```diff
@@ -0,0 +1,24 @@
+package org.dataland.datalandbackend.controller
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.dataland.datalandbackend.interfaces.DataManagerInterface
+import org.dataland.datalandbackend.model.EuTaxonomyDataForNonFinancials
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+/**
+ * Controller for the EU Taxonomy endpoints of non-financial companies
+ * @param myDataManager data manager to be used
+ * @param myObjectMapper object mapper used for converting data classes to strings and vice versa
+ */
+@RequestMapping("/data/eutaxonomy/nonfinancials")
+@RestController
```

Comment:
Discuss route layout

Reply:
Vorschlag:
- Team Discussion nach Daily, in dem du deinen geshelveten Vorschlag zeigst
- Je nach Ausgang der Diskussion => Ticket für die Änderung erstellen

Reply:
Siehe DALA-456